src/assets/scss/components/event.scss

```diff
@@ -0,0 +1,131 @@
+$event-icon-palette: (
+    message: #2274d5,
+    inactive: #919191,
+    default: #57a01f
+    ) !default;
+$event-palette: (
+    primary: #57a01f,
+    active: #404040,
+    inactive: #919191,
+    notification: #d10910,
+    background: #fff,
+    background-inactive: #f8f8f8
+) !default;
+
+$black-shadow: rgba(0, 0, 0, .5);
+
+.footer {
+    text-align: center;
+}
+
+.content {
+    margin-left: auto;
+    margin-right: auto;
+    padding: inherit;
+    white-space: nowrap;
+    @include breakpoint(medium) {
+        width: 80%;
+    }
+}
+
+.icon {
+    background-color: map-get($map: $event-icon-palette, $key: inactive);
+    border: 2px;
+    border-color: map-get($map: $event-icon-palette, $key: inactive);
+    border-radius: 16px;
+    border-style: solid;
+    display: inline-block;
+    height: 32px;
+    vertical-align: top;
+    width: 32px;
+
+    &.small {
+        border-radius: 10px;
+        height: 20px;
+        width: 20px;
+    }
+
+    &.new {
+        background-color: map-get($map: $event-icon-palette, $key: default);
+        border-color: map-get($map: $event-icon-palette, $key: default);
+        @each $name, $color in $event-icon-palette {
+            @if $name != default or $name != inactive {
+                &.#{$name} {
+                    background-color: #{$color};
+                    border-color: #{$color};
+                }
+            }
+        }
+    }
+
+    &.group {
+        background-color: transparent;
+    }
+}
+
+.event-container {
+    h4 {
+        color: map-get($map: $event-palette, $key: inactive);
```

Comment:
tohle je úplně zbytečný kód, kterej není vůbec potřeba, když máš dobře navržený CSS API resp. potřebné util class viz odkaz níže.. ber to jen jako takovej hint, kde je možná cesta..
https://frontstuff.io/in-defense-of-utility-first-css

Reply:
není nutné předělávat ..